A32NX/html_ui/Pages/VCockpit/Instruments/A320_Neo/Map/Svg/A32NX_SvgConstraintElement.js

```diff
@@ -0,0 +1,61 @@
+class A32NXSvgConstraintElement extends SvgConstraintElement {
+    createDraw(map) {
+        const fontSize = map.config.waypointLabelFontSize;
+        let text = "CSTR";
+        let speedText = "";
+        if (this.source) {
+            if (this.source.legAltitude1 >= 10000) {
+                text = "FL" + (this.source.legAltitude1 / 100).toFixed(0);
+            } else {
+                text = (this.source.legAltitude1 / 10).toFixed(0) + "0";
+            }
+        }
+
+        if (this.source.speedConstraint > 10) {
+            speedText = this.source.speedConstraint.toFixed(0) + "KT";
+        }
+        const c = document.createElement("canvas");
+        const ctx = c.getContext("2d");
+        ctx.font = fontSize + "px " + map.config.waypointLabelFontFamily;
+        this._textWidth = Math.max(ctx.measureText(text).width, ctx.measureText(speedText).width);
+        this._textHeight = fontSize * 0.675;
+        this._label = undefined;
+        this.needRepaint = true;
+        const labelId = this.id(map) + "-text-" + map.index;
+        const label = document.getElementById(labelId);
+        if (label instanceof SVGForeignObjectElement) {
+            this._label = label;
+            this.needRepaint = true;
+        }
+        if (!this._label) {
+            this._label = document.createElementNS("http://www.w3.org/2000/svg", "foreignObject");
+            this._label.id = labelId;
+            this._label.setAttribute("width", (this._textWidth + map.config.waypointLabelBackgroundPaddingLeft + map.config.waypointLabelBackgroundPaddingRight).toFixed(0) + "px");
+            this._label.setAttribute("height", (this._textHeight * (this.source.speedConstraint > 0 ? 2.2 : 1) + map.config.waypointLabelBackgroundPaddingTop + map.config.waypointLabelBackgroundPaddingBottom).toFixed(0) + "px");
+            const canvas = document.createElement("canvas");
+            canvas.setAttribute("width", (this._textWidth + map.config.waypointLabelBackgroundPaddingLeft + map.config.waypointLabelBackgroundPaddingRight).toFixed(0) + "px");
+            canvas.setAttribute("height", (this._textHeight * (this.source.speedConstraint > 0 ? 2.2 : 1) + map.config.waypointLabelBackgroundPaddingTop + map.config.waypointLabelBackgroundPaddingBottom).toFixed(0) + "px");
+            this._label.appendChild(canvas);
+            const context = canvas.getContext("2d");
+            if (map.config.waypointLabelUseBackground) {
+                context.fillStyle = "black";
+                context.fillRect(0, 0, this._textWidth + map.config.waypointLabelBackgroundPaddingLeft + map.config.waypointLabelBackgroundPaddingRight, this._textHeight + map.config.waypointLabelBackgroundPaddingTop + map.config.waypointLabelBackgroundPaddingBottom);
+            }
+            context.fillStyle = "magenta";
```

Comment:
if possible, use the color `--displayMagenta: #ff94ff` instead, can be found in the display common.css.

Reply:
Will do thanks

Reply:
@MisterChocker I've changed the color to `#ff94ff` but the numbers now are very hard to read since that color is less vivid, is it this color that is used in the real A320?